src/config/index.js

```diff
@@ -19,7 +19,14 @@ const setConfig = update => {
   return writeJSONSync(configFilePath, mergeDeep(readJSONSync(configFilePath), update))
 }
 
+const isURLValid = () => {
+  const { SWAGGERHUB_URL } = getConfig()
+  return SWAGGERHUB_URL.endsWith('api.swaggerhub.com') ||
+   (!SWAGGERHUB_URL.includes('api.swaggerhub.com') && SWAGGERHUB_URL.endsWith('/v1'))
```

Comment:
unnecessary check here as condition could never be reached if url contains `api.swaggerhub.com`

Reply:
actually - my mistake - nevermind. :)